packages/workflow/src/interfaces.ts

```diff
@@ -135,6 +135,11 @@ export interface WorkflowInfo {
  * Never rely on this information in Workflow logic as it will cause non-deterministic behavior.
  */
 export interface UnsafeWorkflowInfo {
+  /**
+   * Current system time in milliseconds
```

Comment:
I'd explain what the difference between this and `Date.now()` is.
Also wondering if we just want to expose the raw `Date` object instead of just `now()`.

Reply:
Adding: The safe version of time is `new Date()` and `Date.now()`, which are set on the first invocation of a Workflow Task

> Also wondering if we just want to expose the raw Date object instead of just now().

`new workflowInfo().unsafe.Date()` and `workflowInfo().unsafe.Date.now()`

I like the simplicity of just `unsafe.now`, but unsure whether the Date flexiblity is important / if there are things people would need Date for that they can't do `new Date(workflowInfo().unsafe.now)`

Reply:
>Also wondering if we just want to expose the raw Date object instead of just now().

I doubt there is a need for this.

However, if we _really_ want to go there, I think I'd prefer a global `UnsafeDate` class rather than to have something like `new workflowInfo().unsafe.Date()`.

Reply:
Agree, let's keep just `now()`